examples/stacks/lapp-stack/my_app/config.php

```diff
@@ -1,5 +1,5 @@
 <?php
-	$db_hostname	= "127.0.0.1";
+	$db_hostname	= "0.0.0.0";
```

Comment:
unclear if this works, but hardcoding it to `127.0.0.1` certainly does not work in cloud environments